### PR TITLE
update to css

### PR DESCRIPTION
**Pull request by matthewcarleton**
Covers requested changes from @joelanford to website doc styles.

### CODE REVIEW

**Comment by marc-obrien** [comment]
LGTM

File: website/assets/scss/_docs.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        font-size: var(--of--FontSize--sm);
+    }
+    h3 {
+        font-size: var(--of--FontSize--md);
+    }
+    h2 {
+        font-size: var(--of--FontSize--md);
+    }
+    h1 {
+        font-size: var(--of--FontSize--lg);
```
Comment (camilamacedo86) [inline website/assets/scss/_docs.scss:31]: @matthewcarleton, 

I just checked that it broke the left menu. See that it has now a wrong organization. 

After this PR: 

Screenshot 2020-04-24 at 12 54 25

Before and how it should be: 

Screenshot 2020-04-24 at 12 55 44

Would be possible to fix it? 

c/c @asmacdo @jmrodri

Comment (matthewcarleton) [reply website/assets/scss/_docs.scss:31]: Ah yup, I can fix it in my current PR, just didn’t scope the styles to just the content area.

**Pull request merged**